client/src/components/DetailsEvent/index.js

```diff
@@ -1,13 +1,180 @@
-import React from 'react'
-import Header from '../Header/header';
-class DetailsEvent extends React.Component {
+import React, { Component } from "react";
+import NavBar from "../NavBar";
+import SideBar from "../SideBar";
+import axios from "axios";
+
+class DetailsEvent extends Component {
+  state = {
+    id: this.props.history.location.event.id,
+    status: this.props.history.location.event.status,
+    loading: false
+  };
+
+  componentDidMount() {
+    axios
+      .get(`/event/${this.state.id}`)
+      .then(data => {
+        this.setState({
+          event_details: data.data.event,
```

Comment:
seems like maybe call it `apiResponse` on line 16 or deconstruct: `.then({ data } => {` so then you just have to do `data.event` here?